src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,42 @@
+package duke.task;
+import java.time.LocalDate;
```

Comment:
Can consider adding a line break after the package statement (for other classes too), as you've done so in Event.java :)